app.js

```diff
@@ -11,4 +11,24 @@ const {
 app.get('/api/topics', getTopics);
 app.get('/api/articles/:article_id', getArticleByArticleId);
 
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg })
+  } else {
+    next(err)
+  }
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Invalid id" })
+  } else {
+    next(err)
+  }
+});
+
+
+app.use((err, req, res, next) => {
+  res.status(500).send({ msg: "Internal server error" });
+})
```

Comment:
Great error handlers!
I would get a console.log in your 500s one at the bottom. If you haven't seen the error before you need the opportunity to take a look